### PR TITLE
Geomap: Improve tooltip url for photos layer

### LINKED ISSUE #63482: Geomap: Url hyperlink in Photos layer

### PR DESCRIPTION
**Pull request by adela-almasan**
https://user-images.githubusercontent.com/88068998/220323397-348864f8-4ac1-40ad-94ba-434dc13bcbd8.mov

Fixes #63482

### CODE REVIEW

File: public/app/plugins/panel/geomap/components/DataHoverRows.tsx [hunk 1]
```diff
@@ -100,6 +109,6 @@ export const generateLabel = (feature: FeatureLike, idx: number): string => {
 
 const getStyles = (theme: GrafanaTheme2) => ({
   collapsibleRow: css`
-    margin-bottom: 0px;
+    margin-bottom: 0;
```
Comment (nmarrs) [inline public/app/plugins/panel/geomap/components/DataHoverRows.tsx:112]: 😅

File: public/app/plugins/panel/geomap/utils/utils.ts [hunk 2]
```diff
@@ -116,3 +116,10 @@ export const getNextLayerName = (panel: GeomapPanel) => {
 
   return `Layer ${Date.now()}`;
 };
+
+export const isUrl = (url: string) => {
```
Comment (nmarrs) [inline public/app/plugins/panel/geomap/utils/utils.ts:120]: We might want to consider using a built in library function for this so we don't have to own / maintain this 🤔 maybe we can double check in #frontend slack channel if we already have a common way of doing this check?

**Review by nmarrs — APPROVED** [review]
Molto bene 🇮🇹

**Comment by nmarrs** [comment]
Given this layer is in alpha + we probably are not going to promote this layer to general release until 9.5 I set target for 9.5 instead of 9.4.1 👍

File: public/app/plugins/panel/geomap/utils/utils.ts (hunk 2 above)
Comment (adela-almasan) [reply public/app/plugins/panel/geomap/utils/utils.ts:120]: @nmarrs - updated with Gilles' suggestion. I was having some second thoughts this morning about over-complicating the validation.. in the end I don't think we want to validate users' data, but make it clickable if it resemble an url (_http_). If the url is valid or not, might be someone else's problem..

Comment (nmarrs) [reply public/app/plugins/panel/geomap/utils/utils.ts:120]: Yeah that's a good point. If the image shows up or not is validation itself 😅 before we promote this later we may want to consider having someone in the security team test for vulnerabilities given we are displaying images based on user provided data

**Pull request merged**